documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -1805,3 +1806,23 @@ or
 ```
 
 Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [ARM](#arm-violations): [Errors](#arm-errors) or [Warnings](#arm-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
+
+### <a name="R2010" ></a>R2010 LongRunningOperationsOptionsValidator
+
+**Category** : SDK Warning
+
+**Applies to** : ARM and Data Plane OpenAPI(swagger) specs
+
+**Output Message** : A LRO Post operation with return schema must have "x-ms-long-running-operation-options" extension enabled.
+
+**Description** : This is a rule introduced to make the understanding of Long Running Operations more clear. A LRO has the following steps:
+
+1. An Initial call is made to the service.
```

Comment:
@lmazuel would you like to review this?

Reply:
Yes, I'd like to review and maybe re-phrase it a little. Can't today, but tomorrow for sure.

Reply:
DOne